datafusion/sqllogictest/test_files/subquery.slt

```diff
@@ -437,7 +437,7 @@ Projection: t1.t1_id, (<subquery>) AS t2_int
 ------Projection: t2.t2_int
 --------Filter: t2.t2_int = outer_ref(t1.t1_int)
 ----------TableScan: t2
---TableScan: t1 projection=[t1_id]
+--TableScan: t1 projection=[t1_id, t1_int]
```

Comment:
How did this work before (was this a bug)?

Reply:
As far as I know, we cannot generate physical plans for these queries. I think it was a bug, however since it couldn't be tested with data (there is no physical plan generated), it was silent.

Reply:
Ok, thanks for clarifying :)